js/script.js

```diff
@@ -0,0 +1,68 @@
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
+
+const defaultBooks = [
+  {
+    title: 'Lorem, ipsum',
+    author: 'Testeroo Testyy',
+  },
+  {
+    title: 'Lorem, ipsum',
+    author: 'Testeroo Testyy',
+  },
+];
```

Comment:
Great job @mrbhi 🤝

- [ ] Kindly remove this static data from your project and only use data input from the user and rendered to the DOM.

Reply:
Thanks @1GORDON